cl/users/models.py

```diff
@@ -284,6 +280,11 @@ class BackoffEvent(AbstractDateTimeModel):
     next_retry_date = models.DateTimeField(
         help_text="The next retry datetime for exponential backoff events.",
     )
+    event_sub_type = models.SmallIntegerField(
+        help_text="The SES bounce subtype that triggered the object.",
```

Comment:
SNS right?

Reply:
I think the correct description is `SES`, since is the service that composes the message, I'll keep this in mind to be consistent with referring to it.